apiserver/facades/schema.json

```diff
@@ -33683,11 +33683,16 @@
         "Schema": {
             "type": "object",
             "properties": {
+                "Kill": {
```

Comment:
I wonder if we should still keep `Stop` for Pinger to keep backwards compatibility.

Reply:
done in 2749e98dae